CI/before_script.msvc.sh

```diff
@@ -213,20 +213,28 @@ if [ -z $VS_VERSION ]; then
 fi
 
 case $VS_VERSION in
+	15|15.0|2017 )
+		GENERATOR="Visual Studio 15 2017"
+		TOOLSET="vc140"
```

Comment:
No, this variable is intended to be vc140. The only reason it is used is to make sure CMake looks for the Boost v140 binaries instead of trying to look for non-existent v141 binaries. I guess this variable should be renamed to something less misleading.